x-pack/plugins/apm/server/lib/connections/get_connection_stats/get_stats.ts

```diff
@@ -54,114 +56,247 @@ export const getStats = async ({
     offset,
   });
 
-  const response = await apmEventClient.search('get_connection_stats', {
-    apm: {
-      events: [ProcessorEvent.metric],
-    },
-    body: {
-      track_total_hits: true,
-      size: 0,
-      query: {
-        bool: {
-          filter: [
-            ...filter,
-            {
-              exists: {
-                field: SPAN_DESTINATION_SERVICE_RESPONSE_TIME_COUNT,
-              },
-            },
-            ...rangeQuery(startWithOffset, endWithOffset),
-            ...excludeRumExitSpansQuery(),
-          ],
-        },
+  const response = await apmEventClient.search(
```

Comment:
This is setup for a demo, the queries will either be combined into one or refactored.